diff --git a/SPECS/libvirt.spec b/SPECS/libvirt.spec
--- a/SPECS/libvirt.spec
+++ b/SPECS/libvirt.spec
@@ -26,7 +26,7 @@
 %endif
 %define arches_vbox             %{arches_x86}
 %define arches_ceph             %{arches_64bit}
-%define arches_zfs              %{arches_x86} %{power64} %{arm}
+%define arches_zfs              %{arches_x86} %{power64} %{arm} aarch64
 %define arches_numactl          %{arches_x86} %{power64} aarch64 s390x
 %define arches_numad            %{arches_x86} %{power64} aarch64
 %define arches_ch               x86_64 aarch64
@@ -76,14 +76,7 @@
     %define with_storage_gluster 0
 %endif
 
-# On Fedora 43, the 'zfs-fuse' package was removed, but is obtainable via
-# other means. Build the backend, but it's no longer considered to be part
-# of 'daemon-driver-storage'.
-%if 0%{?fedora}
-    %define with_storage_zfs      0%{!?_without_storage_zfs:1}
-%else
-    %define with_storage_zfs      0
-%endif
+%define with_storage_zfs      1
 
 %define with_storage_iscsi_direct 0%{!?_without_storage_iscsi_direct:1}
 # libiscsi has been dropped in RHEL-9
